### DISCUSSION TITLE
I don’t know if it is possible to press ctrl alone to output the output to esc, and ctrl+[other keys] to ctrl. The behavior remains unchanged.

### DISCUSSION THREAD

**Discussion by mortyhuang** [vial-kb/vial-gui#294] [Q&A] [answered]
A similar function is [to_if_alone](https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to-if-alone/) in karabiner-elements

If it can be implemented, how should I configure it

**Comment by xyzz** [comment, marked as answer]
You need to set up mod-tap with MT(MOD_LCTL, KC_ESC)

**Reply by mortyhuang** [reply]
Thank you so much, it seems that I have less than 50% exploration of this firmware.
